tests/sparql/eval/sparql-star-op-3.srj

```diff
@@ -0,0 +1,27 @@
+{ "head": {
+    "vars": [ "x" , "left" , "right" ]
+  } ,
+  "results": {
+    "bindings": [
+      { 
+        "x": { "type": "uri" , "value": "http://example/x7" } ,
+        "left": {
+          "type": "triple" , 
+          "value": {
+            "subject":  { "type": "uri" , "value": "http://example/a" } ,
+            "predicate": { "type": "uri" , "value": "http://example/b" } ,
+            "object":   { "type": "literal" , "datatype": "http://www.w3.org/2001/XMLSchema#integer" , "value": "9" }
+          }
+        } ,
+        "right": {
+          "type": "triple" , 
+          "value": {
+            "subject":  { "type": "uri" , "value": "http://example/a" } ,
+            "predicate": { "type": "uri" , "value": "http://example/b" } ,
+            "object":   { "type": "literal" , "datatype": "http://www.w3.org/2001/XMLSchema#integer" , "value": "123" }
+          }
+        }
+      }
+    ]
```

Comment:
Shouldn't this include results for the following?
```
(solution ?x=http://example/x4
  ?left=<http://example/a> <http://example/b> "123"^^<http://www.w3.org/2001/XMLSchema#integer> .
  ?right=<http://example/c> <http://example/d> "123"^^<http://www.w3.org/2001/XMLSchema#integer> .
 )
```

Reply:
In SPARQL 1.1, IRIs aren't in the operator dispatch table for `<` or `>` which makes comparing IRIs by "less than" an 'error'.

Implementations can (and do!) have that as an extension.

Reply:
Something needs to be added for Embedded Triple compare, then, as objects and predicates are blank nodes or IRIs and comparison would always result in a type error, but it seems that the intention is to compare just the object parts. So, the test for comparing `sparql-compare(PREDICATE(A), PREDICATE(B)) != 0` don't seem to serve any purpose, or should at least allow for the result to be a type error. Same for SUBJECT and OBJECT, recursively.

Reply:
No, I think it's probably okay, but I still have a couple of failing tests. Comparison of IRIs and blank nodes is defined if they are the same terms, just not < or >.